### PR TITLE
12-sec: Add minor fixes related to grammar and typos

### PR DESCRIPTION
**Pull request by andreia-oca**
I have checked the spelling and added correct punctuation to chapter 12 of the book.

### CODE REVIEW

File: book/chapters/12-sec/contents.tex [hunk 1]
```diff
@@ -242,13 +242,13 @@ \subsection{Integritatea datelor}
   \centering
   \def\svgwidth{\columnwidth}
   \includesvg[width=0.9\textwidth]{chapters/12-sec/img/checksum.svg}
-  \caption{Generare și verificare sumă de control}
+  \caption{Generare și verificare sumei de control}
```
Comment (razvand) [inline book/chapters/12-sec/contents.tex:245]:
```suggestion
  \caption{Generarea și verificarea sumei de control}
```

**Review by razvand — APPROVED** [review]
Thanks, looks awesome. I have a minor suggestion, please see that. And use present tense for the commit message (do a `git commit --amend` and `git push --force`).

File: book/chapters/12-sec/contents.tex (hunk 1 above)
Comment (andreia-oca) [reply book/chapters/12-sec/contents.tex]: Done!

File: book/chapters/12-sec/contents.tex [hunk 2]
```diff
@@ -69,9 +69,9 @@ \subsection{Noțiuni de securitate}
...
+În obiectivul său de a obține beneficii proprii, un atacator are nevoie de un mod de acces la un sistem. În zilele noastre, accesibilitatea prin Internet înseamnă că multe sisteme sunt atacabile de la distanță. Țintele de început ale atacatorilor sunt punctele de intrare în sistem adică modurile în care se poate accesa un sistem. Spunem că intrările unui sistem reprezintă \textbf{suprafața de atac} a unui sistem (\textit{attack surface}); un sistem are suprafața de atac cu atât mai mare cu cât are un număr mai mare de intrări sau de intrări cât mai largi. O suprafață de atac mare înseamnă risc de securitate mare. Așa că, unul dintre modurile prin care un apărător protejează sistemul este reducerea suprafeței de atac.
```
Comment (razvand) [inline book/chapters/12-sec/contents.tex:72]:
```suggestion
În obiectivul său de a obține beneficii proprii, un atacator are nevoie de un mod de acces la un sistem. În zilele noastre, accesibilitatea prin Internet înseamnă că multe sisteme sunt atacabile de la distanță. Țintele de început ale atacatorilor sunt punctele de intrare în sistem, adică modurile în care se poate accesa un sistem. Spunem că intrările unui sistem reprezintă \textbf{suprafața de atac} a unui sistem (\textit{attack surface}); un sistem are suprafața de atac cu atât mai mare cu cât are un număr mai mare de intrări sau de intrări cât mai largi. O suprafață de atac mare înseamnă risc de securitate mare. Așa că, unul dintre modurile prin care un apărător protejează sistemul este reducerea suprafeței de atac.
```

**Review by razvand — APPROVED** [review]
E perfect. Am o mică sugestie și apoi poate fi făcut merge.

File: book/chapters/12-sec/contents.tex (hunk 2 above)
Comment (andreia-oca) [reply book/chapters/12-sec/contents.tex]: Done.

**Pull request merged**